### ISSUE TITLE
Rotation response is not correct for Chrome on Android

### ISSUE THREAD

**Issue by scottlittle** [bmoren/p5.vr#10]
Starting in landscape mode, I only get a response when rotating screen about the longer axis of my phone.  There is no response for looking left or right.

**Comment by bmoren** [comment]
can you put this into filmmaking terms for clarity? pan/tilt? Im assuming that you are saying that you get no movement on the pan type of movement but you are getting tilt movement? Are you getting errors in the logs?

**Comment by scottlittle** [comment]
Yes, no response for panning.  I haven't taken a look at the logs yet.  Also, shouldn't there be an alert if I open the sketch in portrait mode? I can open a new issue for that if necessary.

**Comment by bmoren** [comment]
Thanks for the feedback, I likely wont have a chance to look into this for a bit, but I encourage you to do so if you can. I know the android situation on this needs work.

for the alert –  I think its likely better to leave the instruction as what to do in portrait mode up to the developer / project. This way the alert / message / design can be customized. 
you can use `deviceOrientation` to get portrait vs landscape

**Comment by bmoren** [comment]
my suspicion is that it it has to do with this section of the code. My guess is that something changed for android with compassHeading

```javascript
p5.prototype._compassdir = false;
  window.addEventListener('deviceorientation', function(e) {
     if(e.webkitCompassHeading || e.compassHeading) {
       heading = e.webkitCompassHeading || e.compassHeading
      _compassdir = map(heading,0.0,360.0,360.0,0.0);
    }
  })
```
